nifi-mock/src/main/java/org/apache/nifi/util/TestRunner.java

```diff
@@ -1064,4 +1064,6 @@ void assertAttributes(
      * @param eventType Provenance event type
      */
      void assertProvenanceEvent(ProvenanceEventType eventType);
+
```

Comment:
It would be helpful to add a basic Java Doc comment on this method to assist developers in using this method.

Reply:
Good catch. Will do.